testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.IO;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+#endif
+using UnityEngine;
+
+/// <summary>
+/// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+/// when trying to build from Setup() steps in tests.
+/// </summary>
+public static class BuildMultiprocessTestPlayer
+{
+    public const string MultiprocessBaseMenuName = "MLAPI Multiprocess Test";
+    public const string BuildAndExecuteMenuName = MultiprocessBaseMenuName + "/Build - Execute multiprocess tests #%t";
+    public const string MainSceneName = "MultiprocessTestingScene";
+    public static string BuildPath => Path.Combine(Path.GetDirectoryName(Application.dataPath), "Builds/MultiprocessTestBuild");
+
+#if UNITY_EDITOR
+    [MenuItem(MultiprocessBaseMenuName+"/Build Test Player #t")]
+    public static void BuildNoDebug()
```

Comment:
nit
(but nevertheless, "Debug" should be replaced with "Development" term anyway)
```suggestion
    public static void BuildRelease()
```

Reply:
I used "debug" because of the remote debugging option I'm adding while building.
buildOptions |= BuildOptions.AllowDebugging;